content/news/release-20150412.md

```diff
@@ -0,0 +1,41 @@
+We are very pleased to announce the Spring release of OpenRA.  This is a huge step forward for the Dune 2000 mod in particular, but also, behind the scenes, for the Tiberian Sun mod which has received support for some crucial features during this cycle.
+
+This release includes a total of over 1400 changes by 53 contributors, the culmination of five and a half months of relentless work, and with almost 2000 files and more than 210,000 changed lines one of the busiest development cycles to date.
+
+The development focus has been on the Dune 2000 mod, which has made some huge steps forward and contains the most visible changes. A new sidebar, sandworms and carryalls are the big new features, which should make any Dune 2000 player feel right at home in our mod. The other mods were not neglected, though. In Red Alert, the standout feature is the support of playable factions such as Russia, England, France, Ukraine and Germany. In both Red Alert and Tiberian Dawn, new single-player missions, both campaign and stand-alone ones, are available that will keep you occupied.
```

Comment:
"both campaign and stand-alone ones" is a little redundant, and removing it may make the sentence flow a bit better.